content/_code-samples/freeze/py/set_global_freeze.py

```diff
@@ -0,0 +1,29 @@
+from xrpl.clients import JsonRpcClient
+from xrpl.models import AccountSet, AccountSetFlag
+from xrpl.transaction import (safe_sign_and_autofill_transaction,
+                              send_reliable_submission)
+from xrpl.wallet import generate_faucet_wallet
+
+client = JsonRpcClient("https://s.altnet.rippletest.net:51234") # connect to testnet
+
+
+# generate wallet
+sender_wallet = generate_faucet_wallet(client)
+
+
```

Comment:
There are a couple spots where spacing is inconsistent, please use at most one line of space between these definitions.